bsconfig.schema.json

```diff
@@ -166,6 +166,18 @@
                 ]
             }
         },
+        "transforms": {
```

Comment:
We should rename the term `transformPlugin` to just `plugins`. I know this started out as just transforms, but it has already changed into both AST handling _and_ file validation _and_ scope validation. I envision there will be more plugin points in the future, so might as well keep the keyword generic.

Reply:
Done

Reply:
@elsassph you missed the renaming `transforms` in this schema file.

Reply:
Fixed...